gaia-theme.css

```diff
@@ -147,6 +148,7 @@ Gaia theme variables
    ---------------------------------------------------------*/
 
   --text-color: var(--color-gamma);
+  --label-color: #000;
```

Comment:
Is this definition required? It should cascade from the above `:root` definition.